integration-test-common/src/test/java/com/datastrato/gravitino/integration/test/container/ContainerSuite.java

```diff
@@ -208,6 +213,39 @@ public void startMySQLVersion5Container(TestDatabaseName testDatabaseName) {
     }
   }
 
+  public void startPostgreSQLContainer(TestDatabaseName testDatabaseName, PGImageName pgImageName) {
+    if (!pgContainerMap.containsKey(pgImageName)) {
+      synchronized (ContainerSuite.class) {
+        if (!pgContainerMap.containsKey(pgImageName)) {
+          // Start PostgreSQL container
+          PostgreSQLContainer.Builder pgBuilder =
+              PostgreSQLContainer.builder()
+                  .withImage(pgImageName.toString())
+                  .withHostName(PostgreSQLContainer.HOST_NAME)
+                  .withEnvVars(
+                      ImmutableMap.<String, String>builder()
+                          .put("POSTGRES_USER", PostgreSQLContainer.USER_NAME)
+                          .put("POSTGRES_PASSWORD", PostgreSQLContainer.PASSWORD)
+                          .build())
+                  .withExposePorts(ImmutableSet.of(PostgreSQLContainer.PG_PORT))
+                  .withNetwork(network);
+
+          PostgreSQLContainer container = closer.register(pgBuilder.build());
+          container.start();
+          pgContainerMap.put(pgImageName, container);
+        }
+      }
+    }
+    synchronized (PostgreSQLContainer.class) {
+      pgContainerMap.get(pgImageName).createDatabase(testDatabaseName);
+    }
+  }
+
+  public void startPostgreSQLContainer(TestDatabaseName testDatabaseName) {
```

Comment:
So, the default image version of PG is `version_13`?

Reply:
Yeah, that's why we have test class `CatalogPostgreSqlIT`, which use `postgres:13` for `DEFAULT_POSTGRES_IMAGE`, and `CatalogPostgreSqlVersion12IT`, `CatalogPostgreSqlVersion14IT`, `CatalogPostgreSqlVersion15IT`, `CatalogPostgreSqlVersion16IT`, which extends `CatalogPostgreSqlIT` and override `postgreImageName`.